From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@debian.org>
Date: Thu, 29 Sep 2016 10:46:53 +0200
Subject: Split definitions to make omitting tests easier

Cherry-picking commit d512d22.

Conflicts:
	Makefile.am
	lrmd/Makefile.am

Gbp-Pq: Topic from-upstream
---
 Makefile.am      | 5 ++++-
 lrmd/Makefile.am | 5 ++++-
 2 files changed, 8 insertions(+), 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 7edb6d5..3abcafd 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -24,12 +24,15 @@ MAINTAINERCLEANFILES    = Makefile.in aclocal.m4 configure DRF/config-h.in \
                         DRF/stamp-h.in libtool.m4 ltdl.m4 libltdl.tar
 
 CORE	= $(LIBLTDL_DIR) replace include lib mcp attrd pengine cib crmd fencing lrmd tools xml
-SUBDIRS	= $(CORE) cts extra doc
+SUBDIRS	= $(CORE) extra doc
 
 doc_DATA = AUTHORS COPYING COPYING.LIB
 
 ACLOCAL_AMFLAGS  = -I m4
 
+# Test components
+SUBDIRS	+= cts
+
 testdir			= $(datadir)/$(PACKAGE)/tests/
 test_SCRIPTS		= coverage.sh BasicSanity.sh
 test_DATA		= valgrind-pcmk.suppressions
diff --git a/lrmd/Makefile.am b/lrmd/Makefile.am
index c20ecd9..d61a20c 100644
--- a/lrmd/Makefile.am
+++ b/lrmd/Makefile.am
@@ -18,7 +18,10 @@
 include $(top_srcdir)/Makefile.common
 
 lrmdlibdir	= $(CRM_DAEMON_DIR)
-lrmdlib_PROGRAMS	= lrmd lrmd_test lrmd_internal_ctl
+lrmdlib_PROGRAMS	= lrmd lrmd_internal_ctl
+
+# Test components
+lrmdlib_PROGRAMS	+= lrmd_test
 
 testdir			= $(datadir)/$(PACKAGE)/tests/lrmd
 test_SCRIPTS		= regression.py
